src/evergreen/evg_task_history.rs

```diff
@@ -0,0 +1,331 @@
+//! Lookup the history of evergreen tasks.
+
+use anyhow::Result;
+use async_trait::async_trait;
+use chrono::{DateTime, Duration, Utc};
+use evg_api_rs::models::stats::{EvgTestStats, EvgTestStatsRequest};
+use evg_api_rs::EvgApiClient;
+use std::collections::HashMap;
+use std::fmt::{Display, Formatter};
+use std::sync::Arc;
+
+const HOOK_DELIMITER: char = ':';
+
+/// Runtime information of hooks that ran in evergreen.
+#[derive(Debug, Clone)]
+pub struct HookRuntimeHistory {
+    /// Name of test that hook ran with.
+    pub test_name: String,
+    /// Name of hook.
+    pub hook_name: String,
+    /// Average runtime of hook.
+    pub average_runtime: f64,
+}
+
+impl Display for HookRuntimeHistory {
```

Comment:
The `Display` trait is similar to overriding `__str__` in a python class. It allows you to customize a printable version of the object.